sky/utils/rich_utils.py

```diff
@@ -15,7 +15,7 @@ class _NoOpConsoleStatus:
     """An empty class for multi-threaded console.status."""
 
     def __enter__(self):
-        pass
+        return self
```

Comment:
Q: which call path into this caused the original issue?

Reply:
The ones like this: https://github.com/skypilot-org/skypilot/blob/f2cf0329ed7624624bf89d49637d498664cbbbe8/sky/provision/provisioner.py#L352-L353

The `status` will be None, when the provisioner is called with `sky_logging.is_silent()`, which is used for `sky spot queue`: https://github.com/skypilot-org/skypilot/blob/f2cf0329ed7624624bf89d49637d498664cbbbe8/sky/cli.py#L1620-L1623

Reply:
SG